src/_product-updates/2025-09-15-custom-meta-tags.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Custom meta tags 
+tags: [New]
+---
+Custom meta tags allow you to add your own `meta` tags to the `head` of your documentation, for example, to enhance your own global search experience with filtering through facets based on these meta tags. 
+
+The `x-metaTags` array is composed of objects made of two elements:
+- `name`: the value that will be given to the name attribute,
+- `content`: the value that will be given to the content attribute.
+
+Adding the `x-metaTags` to the root of your definition:
+```yaml
+x-metaTags:
+  - name: First meta tag
+    content: First tag content
+  - name: Second meta tag
+    content: Second meta tag content
+```
+
+Will apply the tags in the HTML of your documentation page:
+```html
+<head>
+  <meta name="First meta tag" content="First meta tag content" />
+  <meta name="Second meta tag" content="Second meta tag content" />
+</head>
+````
+
+You can apply it at different levels of your API definition: root, topic, operation, ... [Learn more about it in our help center](/help/specification-support/meta-tags/)
```

Comment:
Don't you want specify that root tags are cumulatives to end point and operation tags ?

Reply:
I would expose the exhaustive list of concerned nodes: 

- `root` document
- `topics`
- `x-model`
- `operations`, `webhook`, `operations`, `messages`
- `bindings`,
- `tags` (used for `endpoints`, `webhooks`, `channels` grouped by tag)

But if you think, it's not relevant  forget this comment